modules/ztp-install-openshift-pipelines.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="installing-the-openshift-pipelines-operator_{context}"]
+= Installing the OpenShift Pipelines Operator
+
+Follow this guidance to install the OpenShift Pipelines Operator that is used to run the pipeline.
+
+.Prerequisites
+
+* An installed {product-title} hub cluster.
+* Install the OpenShift CLI (`oc`).
+* Access to the cluster as a user with the `cluster-admin` role.
+* Install `git`. For guidance on installing `git`, see link:https://github.com/git-guides/install-git[Install Git].
+
+.Procedure
+
+. Export the `KUBECONFIG` environment variable:
++
+[source,terminal]
+----
+$ export KUBECONFIG=<path_to_kubeconfig>/kubeconfig
+----
+
+. Run the following bash script `bootstrap.sh` with the `KUBECONFIG` as a parameter to install the OpenShift Pipelines Operator:
++
+[source,terminal]
+----
+$ curl -sLK https://raw.githubusercontent.com/rh-ecosystem-edge/ztp-pipeline-relocatable/main/pipelines/bootstrap.sh | bash -s -- ${KUBECONFIG}
```

Comment:
I guess I felt it was integral as this is the preliminary step preparing the environment, so you can run the pipelines. From a docs perspective I was simply explaining what this step does and since this in part exposed the GitHub URL i thought it was OK to mention it in the description. I am not clear on "Why can't the contents of the repo be reflected in our docs?" Do you mean simply paste the `bootstrap.sh` in the docs.

Reply:
> We should NEVER do this; because the content here; isn't RCM or release controlled by Red Hat; and thus can't be supported as distributed by Red Hat.

Thanks for the feedback, @sferich888 I will kindly ask you to phrase comments like this a bit differently in the future (not using caps would be a good start). We have been hard at work on these docs and it's becoming a bit frustrating at this point.  

> Engineering should move this file to an image or some other distribution method.

These scripts are community-supported, therefore we will not be packaging downstream for the time being. The main goal of this PR is to document the steps and requirements to get a ZTPFW setup. We could unpack this section and document all the steps taken in that `bootstrap.sh` script. It will make the documentation significantly longer and more complex. I fear that doing so may also distract readers from more relevant sections.

Is it possible (I think I read this somewhere in the PR) to mention that these scripts are not supported and that they are only being used as an example?